islandora_xmlsitemap.install

```diff
@@ -13,7 +13,83 @@ function islandora_xmlsitemap_uninstall() {
     'islandora_xmlsitemap_number_of_pids_to_process',
     'islandora_xmlsitemap_last_modified_value',
     'islandora_xmlsitemap_last_modified_field',
-    'islandora_namespace_restriction_enforced',
   );
   array_walk($variables, 'variable_del');
 }
+
+/**
+ * Implements hook_install().
+ */
+function islandora_xmlsitemap_install() {
+  // XXX: Taking advantage of the update hook's batch functionality to run the
+  // same batch on install as one would have to in the update hook.
+  islandora_xmlsitemap_update_7000();
+}
+
+/**
+ * Implements hook_schema().
+ */
+function islandora_xmlsitemap_schema() {
+  return array(
+    'islandora_xmlsitemap_entry_ids' => array(
+      'description' => 'Autoincrementing table to store IDs we can use for xmlsitemap table entries.',
+      'fields' => array(
+        'id' => array(
+          'description' => 'Current ID number',
+          'type' => 'serial',
+          'not null' => TRUE,
+        ),
+        'pid' => array(
+          'description' => 'The PID relating to this ID.',
+          'type' => 'varchar',
+          'length' => 255,
+        ),
+      ),
+      'primary key' => array('id'),
+    ),
+  );
+}
+
+/**
+ * Add the ID table schema in.
+ *
+ * XXX: satisfies a race condition (details in the @see below); the best way to
```

Comment:
The comments on update hooks become UI elements and should be written from that perspective. This is user documentation.  Any development documentation and syntax should be moved inside the function.